Europe/es_ES.json

```diff
@@ -0,0 +1,261 @@
+{
+  "Garage": "Garage",
+  "Settings": "Configuraciones",
+  "Admin_Panel": "Panel de Administración",
+  "Logout": "Salir",
+  "Dark_Mode": "Modo oscuro",
+  "Enable_CSV_Imports": "Activar importaciones CSV",
+  "Use_Imperial_Calculation_for_Fuel_Economy_Calculations(MPG)": "Utilice el cálculo imperial para los cálculos de ahorro de combustible (MPG)",
+  "This_Will_Also_Change_Units_to_Miles_and_Gallons": "Esto también cambiará las unidades a millas y galones",
+  "Use_UK_MPG_Calculation": "Utilizar el cálculo de MPG del Reino Unido",
+  "Input_Gas_Consumption_in_Liters,_it_will_be_converted_to_UK_Gals_for_MPG_Calculation": "Consumo de combustible de entrada en litros, se convertirá a galones UK para el cálculo de MPG",
+  "Sort_lists_in_Descending_Order(Newest_to_Oldest)": "Ordenar listas en orden descendente (de más reciente a más antiguo)",
+  "Replace_$0.00_Costs_with_---": "Sustituir costes a 0,00€ por ---",
+  "Use_Three_Decimals_For_Fuel_Cost": "Utilice tres decimales para el coste del combustible",
+  "Display_Saved_Notes_in_Markdown": "Mostrar notas guardadas en Markdown",
+  "Auto_Refresh_Lapsed_Recurring_Reminders": "Actualización automática de recordatorios recurrentes caducados",
+  "Auto_Insert_Odometer_Records": "Inserción automática de registros de cuentakilómetros",
+  "Only_when_Adding_Service/Repair/Upgrade/Fuel_Record_or_Completing_a_Plan": "Solamente cuando se añade un servicio/reparación/actualización/registro de combustible o se completa un plan.",
+  "Enable_Authentication": "Activar autenticación",
+  "Visible_Tabs": "Pestañas visibles",
+  "Service_Records": "Registros de servicio",
+  "Dashboard": "Cuadro de mandos",
+  "Repairs": "Reparaciones",
+  "Upgrades": "Actualizaciones",
+  "Fuel": "Combustible",
+  "Odometer": "Cuentakilómetros",
+  "Taxes": "Impuestos",
+  "Notes": "Notas",
+  "Reminder": "Recordatorio",
+  "Supplies": "Suministros",
+  "Planner": "Planificador",
+  "Default_Tab": "Pestaña predeterminada",
+  "Service_Record": "Hoja de servicios",
+  "Tax": "Impuesto",
+  "Reminders": "Recordatorios",
+  "Backups": "Copias de seguridad",
+  "Make": "Haz",
```

Comment:
this has already being merged in the main repo, but this translation is used in two different places with different meanings in languages like spanish (car manufacturer and create backup button)
consider changing the translation

Reply:
I am sorry to say that I do not understand this point.

Can you reference the line number that you think is conflicting?

Reply:
Line 37. 
I think it's already changed in the en_us original file, but this translation used to be used in different locations with different meanings, one being the `car manufacturer` ("marca o fabricante"), and the other one being the `create backup button` ("hacer").
As stated [here](https://github.com/hargata/lubelog_translations/pull/10#issuecomment-1937079301) "Make" is now used just to refer to the car manufacturer and "Create" (new translation) is used to refer to the `create backup button`.

So, this one should be changed to something like "fabricante".

Reply:
ok, fixed.